File: SPA.Server/Models/Worker.cs
using System;
using System.Collections.Generic;

namespace SPA.Server.Models
{
    public partial class Worker
    {
        public Worker()
        {
            Term = new HashSet<Term>();
            WorkerRole = new HashSet<WorkerRole>();
        }

        public int WorkerId { get; set; }
        public int PersonId { get; set; }
        public DateTime? DateFrom { get; set; }
        public DateTime? DateTo { get; set; }

        public Person Person { get; set; }
        public ICollection<Term> Term { get; set; }
        public ICollection<WorkerRole> WorkerRole { get; set; }
    }
}

File: SPASaloon/src/app/api/models/worker.ts
/* tslint:disable */
import { Person } from './person';
import { Term } from './term';
import { WorkerRole } from './worker-role';
export interface Worker {
  workerId?: number;
  personId?: number;
  dateFrom?: string;
  dateTo?: string;
  person?: Person;
  term?: Array<Term>;
  workerRole?: Array<WorkerRole>;
}
